PoshSSDTBuildDeploy/Functions/PublishDatabaseDeployment.ps1

```diff
@@ -1,5 +1,5 @@
 Function Publish-DatabaseDeployment {
-
+    [cmdletbinding()]
```

Comment:
Added this so you can use verbose switch at a higher level and removed the explicit verbose switches below.